Weather/Views/TodayWeatherView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  TodayWeatherView.swift
+//  Weather
+//
+//  Created by Oleshkevich Artem on 26.07.22.
+//
+
+import SwiftUI
+
+struct TodayWeatherView: View {
+    @ObservedObject var cityViewModel: CityViewViewModel
+    
+    var body: some View {
+        VStack(spacing: 10) {
+            Text(NSLocalizedString("Today", comment: ""))
+                .font(.largeTitle)
+                .bold()
+                .shadow(color: .black, radius: 1)
+            
+            HStack(spacing: 20) {
+                LottieView(name: cityViewModel.getLottiesAnimationFor(icon: cityViewModel.weatherIcon))
+                    .frame(width: 100, height: 100)
+                
+                VStack(alignment: .leading) {
+                    Text("\(cityViewModel.temperature)℃")
+                        .font(.system(size: 42))
+                        .shadow(color: .black, radius: 1)
+                    Text(NSLocalizedString(cityViewModel.conditions, comment: ""))
+                        .shadow(color: .black, radius: 1)
+                }
+            }
+        
+            HStack {
+                Spacer()
+                widgetView(image: "wind", color: .gray, title: "\(cityViewModel.windSpeed)" + NSLocalizedString("Wind speed unit", comment: ""))
```

Comment:
Please, use localisation with parameters in this case because in some languages the order of numbers and abreviation can be different. The localizable file should have such key: `"windSpeedUnit %@" = "%@ mp/h`. And you should use it like this: Text("windSpeedUnit \(cityViewModel. windSpeed)")